server/drivers/clickhouse/_clickhouse.js

```diff
@@ -58,6 +58,13 @@ async function handleStatementAndGetMore(results, statement, config) {
     // but error.message contains the detail on syntax issue
     return Promise.reject(new Error(await statement.error));
   }
+  try {
+    statement = JSON.parse(statement);
+  } catch (e) {
+    // Queries like 'insert/create table/drop table' SQLs have empty response body,
+    // so it return the success message.
+    return Promise.reject(new Error(statement || 'Successfully Executed.'));
```

Comment:
I think we should reserve errors for query failures - What do you think about returning an empty response or a message stubbed in as a response?

Reply:
Yeah, you are right. We can return the original response,  because it's not the query failure.

Reply:
I took a look at clickhouse/index.js and I think if this returns `{ data: [], rows: [] }` or something similar if you want to stub in a single column with a single row value it should be good

Reply:
En. Now I modify the code, it return the SUCCESS with the `{ data: [], columns: [] }` format when response is ok and body is empty.